inspire/base/static/less/accounts/settings/account-settings.less

```diff
@@ -0,0 +1,65 @@
+/**
+ * -*- mode: text; coding: utf-8; -*-
+ *
+ * This file is part of INSPIRE.
+ * Copyright (C) 2015 CERN.
+
+ * INSPIRE is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 2 of the
+ * License, or (at your option) any later version.
+ *
+ * Invenio is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Invenio; if not, write to the Free Software Foundation, Inc.,
+ * 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+ */
+
+ @import (reference) "../../inspire.less";
+
+/* Account settings page style */
+
+.list-group-item {
+	cursor: pointer;
+}
+
+.link-display {
+	display: block;
+}
+
+/*
+.settings-form {
+	width:95%; 
+	margin-left:20px;
+}
+*/
+.list-group-item.active.custom-active {
+    background-color: #EBEBEB;
+    border-color: #EBEBEB;
+}
+
+.list-group-item.active.custom-active:hover {
+    background-color: #E3E3E3;
+}
+
+#top-row {
+	margin: 10px 0px 30px 0px;
+}
+
+#user-icon {
+	float:left;
+}
+
+.panel > .panel-heading {
```

Comment:
Even though this file is ```account-settings.less``` it seems like it will be added to all pages https://github.com/inspirehep/inspire-next/pull/246/files#diff-1e868292033d02e4d918047922954ccaR33 and it is modifying global Bootstrap classes like ```.panel > .panel-heading``` or ```.list-group-item```.

If we want these changes to apply only to the accounts page then we should have a custom class to make sure it does not affect the whole site.

If the changes are meant to happen across the site then you can add the `less` rules into the appropriate files:

1. ```.panel > .panel-heading``` in ```less/base/panels.less```

2. ```.list-group-item``` you can create in the same folder a file ```list-group.less``` where we introduce the modifications to the Bootstrap list-group.less.

The general idea when you modify a Bootstrap class globally for the site is that you search in ```vendors/bootstrap/less``` to see what file contains the ```less``` rules you want to modify, then add/create to the equivalent in ```inspire/less/base```.